ably/ably_test.go

```diff
@@ -132,6 +132,27 @@ func assertNotDeepEquals(t *testing.T, expected interface{}, actual interface{})
 	}
 }
 
+func assertSubset(t *testing.T, set []string, subset []string) {
+	t.Helper()
+	for _, item := range subset[1:] {
```

Comment:
What's special about `subset[0]`? It seems to me that the `[1:]` should be in the caller, not here.

Reply:
Thanks, fixed it !